Snippets/Sqs/Sqs_All/QueueDeletion/QueueDeletionUtils.cs

```diff
@@ -0,0 +1,99 @@
+﻿namespace SqsAll.QueueDeletion
+{
+    using System;
+    using System.Threading.Tasks;
+    using Amazon.Runtime;
+    using Amazon.SQS;
+    using Amazon.SQS.Model;
+    using SqsAll.QueueCreation;
+    using Sqs_All;
+
+    #region sqs-delete-queues
+
+    public static class QueueDeletionUtils
+    {
+        public static async Task DeleteAllQueues()
+        {
+            using (var client = ClientFactory.CreateSqsClient())
+            {
+                await DeleteQueues(client)
+                    .ConfigureAwait(false);
+            }
+        }
+
+        static async Task DeleteQueues(IAmazonSQS client)
+        {
+            var queuesToDelete = await client.ListQueuesAsync("")
```

Comment:
This deletes every queue in the region, may be a bit heavy-handed :) The queue name prefix is typically used to isolate application environments in the same region (e.g., "DEV", "STAGING", "PROD", etc). 

Perhaps allow the queue name prefix to be passed in as a parameter?

(Unfortunately this suggestion cross-cuts pretty much everything in this PR - sorry :P)